cloudify_docker_image/make_cfy_manager_image.sh

```diff
@@ -11,17 +11,25 @@ CFY_RPM="cloudify-manager-install.rpm"
 CONTAINER_NAME="cfy_manager"
 BASE_IMAGE="cfy_manager_base"
 IMAGE_PUB_NAME="docker_cfy_manager"
+declare -a IMAGE_TAGS=( "latest" "centos7_v1.0_manager_v4.3" )
 DOCKER_RUN_FLAGS="--name ${CONTAINER_NAME} -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run
  --tmpfs /run/lock --security-opt seccomp:unconfined --cap-add SYS_ADMIN"
 MANAGER_CONFIG_LOCATION="/opt/cloudify"
+DOCKER_REPO="manager_images"
+
+if [ -z "$DOCKER_ID_USER" ] || [ -z "$DOCKER_ID_PASSWORD" ];
+  then
```

Comment:
please note that this script runs with a `set -u`, so this will only be executed if the env vars are set to an empty string. If the env vars aren't set at all, instead a bash error message will be shown: `script.sh: line 20: DOCKER_ID_USER: unbound variable`.

To show this error message also when the env vars aren't set at all, wrap this check with `set +u` (and `set -u` after).

But IMO, the `-u` and the default bash message is enough, and we don't need this check.

Reply:
Fixed. Thanks again!

Reply:
Fixed. Thanks again!